v1/pom.xml

```diff
@@ -218,7 +215,7 @@
       <version>${hadoop.version}</version>
       <exclusions>
         <exclusion>
-          <groupId>org.codehaus.jackson</groupId>
+          <groupId>com.fasterxml.jackson</groupId>
```

Comment:
This dependency moved and all the versions on the old naming have the vulnerability, so moving to the new repository name.